wowup-electron/system-tray.ts

```diff
@@ -33,6 +34,12 @@ export function createTray(
         }
       },
     },
+    {
```

Comment:
lets just comment this out for now, since its more complex

Reply:
Done